### ISSUE TITLE
[Bug]: Sort - `buffer` and `digraphs` sources do not have `sortText` keys

### ISSUE THREAD

**Issue by kunzaatko** [Saghen/blink.cmp#596]
### Make sure you have done the following

- [X] I have updated to the latest version of `blink.cmp`
- [X] I have read the README

### Bug Description

When including the one of the sources `digraphs` / `buffer` the sort function throws comparison with `nil` value error. This was introduced by the commit `76230d5` and comes from the function `sort.sort_text` in [`lua/blink/cmp/fuzzy/sort.lua`](https://github.com/saghen/blink.cmp/blob/d6821651b145c730ca59faee638947a067243b24/lua/blink/cmp/fuzzy/sort.lua#L31-L32).
The tables `a` / `b` for `digraphs` looks like: 
```lua
{
  cursor_column = 7,
  filterText = "0a",
  kind = 10,
  label = "0a ۰",
  labelDetails = {
    detail = "U+06F0"
  },
  score = 9,
  score_offset = 1,
  source_id = "digraphs",
  source_name = "digraphs",
  textEdit = {
    newText = "۰",
    range = {
      ["end"] = {
        character = 7,
        line = 65
      },
      start = {
        character = 5,
        line = 65
      }
    }
  }
}
```
and for `buffer` like
```lua
{
  cursor_column = 6,
  insertText = "Config",
  insertTextFormat = 1,
  kind = 1,
  label = "Config",
  score = 6,
  score_offset = -2,
  source_id = "buffer",
  source_name = "Buffer"
}
```
and the error thrown is:
```
Error executing vim.schedule lua callback: ...l/share/nvim/lazy/blink.cmp/lua/blink/cmp/fuzzy/sort.lua:32: attempt to compare nil with string
stack traceback:
        ...l/share/nvim/lazy/blink.cmp/lua/blink/cmp/fuzzy/sort.lua:32: in function 'sorting_func'
        ...l/share/nvim/lazy/blink.cmp/lua/blink/cmp/fuzzy/sort.lua:13: in function <...l/share/nvim/lazy/blink.cmp/lua/blink/cmp/fuzzy/sort.lua:11>
        [C]: in function 'sort'
        ...l/share/nvim/lazy/blink.cmp/lua/blink/cmp/fuzzy/sort.lua:11: in function 'fuzzy'
        ...re/nvim/lazy/blink.cmp/lua/blink/cmp/completion/list.lua:109: in function 'fuzzy'
        ...re/nvim/lazy/blink.cmp/lua/blink/cmp/completion/list.lua:85: in function 'show'
        ...re/nvim/lazy/blink.cmp/lua/blink/cmp/completion/init.lua:34: in function <...re/nvim/lazy/blink.cmp/lua/blink/cmp/completion/init.lua:29>
```

### neovim version

NVIM v0.10.2 Build type: Release LuaJIT 2.1.1713484068 Run "nvim -V1 -v" for more info

### `blink.cmp` version: branch, tag, or commit

d6821651b145c730ca59faee638947a067243b24

**Comment by Saghen** [comment]
Duplicate of #595

**Comment by kunzaatko** [comment]
This was probably fixed in "3ba583c"
